### PR TITLE
update welcome screen

### PR DESCRIPTION
**Pull request by robinschmid**
image

### CODE REVIEW

**Comment by robinschmid** [comment]
# Old screen 
image

File: src/main/java/io/github/mzmine/gui/mainwindow/introductiontab/MZmineIntroduction.html [hunk 1]
```diff
@@ -32,61 +32,66 @@
...
   <b>Feature list methods</b>. Use the <b>Visualization</b> modules to explore the results. If
   you have a trouble with any module click on the <b>Help</b> button in the corresponding parameter
-  setup
-  dialog. More information, tutorials, and videos are available at the MZmine website.
+  setup dialog. More information, tutorials, and videos are available at the MZmine website.
 </p>
 
 </body>
 </html>
+</body>
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/gui/mainwindow/introductiontab/MZmineIntroduction.html:97]: end line

File: src/main/java/io/github/mzmine/gui/mainwindow/introductiontab/MZmineIntroduction_darkmode.html [hunk 2]
```diff
@@ -32,60 +32,65 @@
 
 <body text="white" style="background-color: #151924">
 
-<center><img src="MZmine_logo_RGB.png" align="BOTTOM" style="height:20%;"></center>
+<!-- logo is different in dark and light mode -->
+<center><h1><img src="MZmine_logo_white.png" align="BOTTOM" style="height:20%;"></h1></center>
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/gui/mainwindow/introductiontab/MZmineIntroduction_darkmode.html:36]: still have the peaks in the rgb colors?

Comment (robinschmid) [reply src/main/java/io/github/mzmine/gui/mainwindow/introductiontab/MZmineIntroduction_darkmode.html:36]: Currently thats the design of the logo
I can Try some stuff

**Pull request merged**